hawc/apps/lit/filterset.py

```diff
@@ -188,6 +214,52 @@ def filter_anything_tagged_me(self, queryset, name, value):
         ).filter(my_tag_count__gt=0)
         return queryset.distinct()
 
+    def filter_tag_additions(self, queryset, name, value):
```

Comment:
These methods look good, but they could be slow. I don't think we need to change anything now.

We may need to monitor performance with some real data after its in the wild for a while to determine if we need to optimize.  I'm thinking specifically of a case where we include descendants, and the parent has many children. I wonder if there's a way to see to see if a reference has any of these tags in a single query.

I think you could do it it with the ArrayField using [overlap](https://docs.djangoproject.com/en/4.2/ref/contrib/postgres/fields/#overlap), but that's only for the deletions, not the additions.